butane_core/Cargo.toml

```diff
@@ -52,3 +52,10 @@ fake = { version = "2.5.1-alpha.1", registry = "franklin-ai-rosalind", features
   "uuid",
 ] }
 rand = "0.8"
+utoipa = { version = "=3.0.4-alpha.1", registry = "franklin-ai-rosalind", features = [
+  "actix_extras",
```

Comment:
do we need all these features for butane ?

Reply:
Good point, all we need here is the ToSchema trait.